packages/server/src/utilities/fileSystem/clientLibrary.ts

```diff
@@ -120,7 +121,12 @@ export async function updateClientLibrary(appId: string) {
       ContentType: "application/javascript",
     }
   )
-  await Promise.all([manifestUpload, clientUpload])
+
+  const manifestSrc = fs.promises.readFile(manifest, "utf8")
+
+  await Promise.all([manifestUpload, clientUpload, manifestSrc])
+
+  return JSON.parse(await manifestSrc)
```

Comment:
Returning the manifest from both `revertClientLibrary` and `updateClientLibrary` so that the controllers that consume these can check the manifest for if the user's client version has skeleton loaders enabled.